modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/styles/_data_set_views.scss

```diff
@@ -0,0 +1,8 @@
+.data-views-list-sheet {
+
+    .sheet-header{
+
+    border-bottom: 1px solid $gray-200;
+    margin-bottom: map-get($spacers, 4);
+    }
+}
```

Comment:
Couple of things here:
- SF is clearly off. Please run `gradle formatSource`.
- If there is only one child in SCSS, we can simplify this to `.data-views-list-sheet .sheet-header {`
- If you are using standard margin or padding, you can use `m` and `p` CSS classes. `margin-bottom` style in PR would be `mb-4`. This is optional, some people prefer styles over CSS classed. 
- If you end up with styles specific to the `list` view, let's add them to `/styles/views/_list.scss`. You can add something like `.list-view` class in the root of the `List`. We can refactor `table` view styles to this pattern later on.

Reply:
Refactor done :+1: